consensus/core/src/authority_node.rs

```diff
@@ -264,9 +265,10 @@ where
             dag_state.clone(),
             commit_consumer.block_sender.clone(),
         );
+        // TODO(fastpath): recover incrementally by chunk inside CommitObserver::new()
         transaction_certifier.recover(
             block_verifier.as_ref(),
-            commit_consumer.last_processed_commit_index,
+            commit_consumer.replay_after_commit_index,
```

Comment:
`replay_after_commit_index` here is equivalent to `last_processed_commit_index` before. `last_processed_commit_index` has been used as the "replay after" index. The actual `last_processed_commit_index` from consensus handler has not been passed into consensus. This change passes in both parameters and names them properly.